qutebrowser/commands/command.py

```diff
@@ -173,6 +179,11 @@ def _check_prerequisites(self, win_id):
                 "{}: Only available with {} "
                 "backend.".format(self.name, self.backend.name))
 
+        if count == 0 and not self._zero_count:
+            raise cmdexc.PrerequisitesError(
+                "{}: Argument zero_count not been set to true!".format(
+                    self.name))
```

Comment:
This is the error displayed to the user, so this would be a confusing output. I think something like "{}: A zero count is not allowed for this command!" would be better.